src/check_subid_range.c

```diff
@@ -36,20 +36,20 @@ main(int argc, char **argv)
 	log_set_logfd(stderr);
 
 	if (argc != 5)
-		exit(1);
+		return 1;
 
 	owner = argv[1];
 	check_uids = argv[2][0] == 'u';
 	if (get_uid(argv[3], &start) == -1)
-		exit(1);
+		return 1;
 	if (str2ul(&count, argv[4]) == -1)
-		exit(1);
+		return 1;
 	if (check_uids) {
 		if (have_sub_uids(owner, start, count))
 			exit(0);
-		exit(1);
+		return 1;
```

Comment:
For error handling, I very much prefer exit(3).

For successful exiting, I still have some preference for exit(3), but could be convinced of using return.
I think exit(3) has more readability, as it tells that the program is over, while with return you have to remember (and check) that you're in main().

BTW, right here there's a successful exit(3) call unchanged.  :)

Reply:
I think we can close this PR. I'm not fully convinced but wanted to offer in case someone really wants this. So ... No. :)